components/member-profile/member-profile.module.scss

```diff
@@ -27,6 +27,7 @@
   padding: 20px 30px 20px;
   background: white;
   box-shadow: 0 0 15px -7px rgba(0, 0, 0, 0.65);
+  width: 40vw;
```

Comment:
can we please change this to `%` and make sure it's working correctly? it's recommended to use `%` instead of `vh & vw`

Reply:
> can we please change this to `%` and make sure it's working correctly? it's recommended to use `%` instead of `vh & vw`

yep

Reply:
> can we please change this to `%` and make sure it's working correctly? it's recommended to use `%` instead of `vh & vw`

% will change the width of contribution container according to the number of badges
![Screenshot (10)](https://user-images.githubusercontent.com/61655940/110840641-fc21a480-82ca-11eb-90a3-245ba7486273.png)
![Screenshot (11)](https://user-images.githubusercontent.com/61655940/110840645-fd52d180-82ca-11eb-99c1-bfd93725bc5e.png)

Reply:
> can we please change this to `%` and make sure it's working correctly? it's recommended to use `%` instead of `vh & vw`

% will change the width of contribution container according to the number of badges
![Screenshot (10)](https://user-images.githubusercontent.com/61655940/110840641-fc21a480-82ca-11eb-90a3-245ba7486273.png)
![Screenshot (11)](https://user-images.githubusercontent.com/61655940/110840645-fd52d180-82ca-11eb-99c1-bfd93725bc5e.png)